tests/Sdk/Unit/Trace/PropagationMapTest.php

```diff
@@ -125,4 +125,20 @@ public function testGetArrayValuesFromCarrier()
         $this->assertSame('alpha', $map->get($carrier, 'a'));
         $this->assertSame('bravo', $map->get($carrier, 'b'));
     }
+
+    /**
+     * @test
+     */
+    public function testGetNumericalKeyFromCarrier()
+    {
+        // Carrier contains an array as one of the values
+        $carrier = [
+            1 => ['alpha'],
+            'b' => 'bravo',
+        ];
+        var_dump($carrier);
```

Comment:
May want to remove these `var_dump` instances.

Reply:
Yeah just saw that. It's already out 👍🏾